pxr/pxrConfig.cmake.in

```diff
@@ -47,6 +47,16 @@ if(@PXR_ENABLE_PYTHON_SUPPORT@)
     endif()
 endif()
 
+include(CMakeFindDependencyMacro)
+
+if(@PXR_FIND_TBB_IN_CONFIG@)
```

Comment:
Can we replace this with:

```
if (NOT DEFINED PXR_FIND_TBB_IN_CONFIG)
    set(PXR_FIND_TBB_IN_CONFIG "@PXR_FIND_TBB_IN_CONFIG@")
endif()
if(PXR_FIND_TBB_IN_CONFIG)
```

I know there's already an established precedent of just doing the "bare" `if(@PXR_VAR@)`, but I feel this makes the resulting `pxrConfig.cmake` less readable, and prevents downstream users from being able to check (or even override) `PXR_FIND_TBB_IN_CONFIG` directly themselves.

Reply:
I agree that lines like `if(OFF)` or even `if()` are cryptic. I like your suggestion but why do we need to check if it's defined, why not just setting it?

```cmake
set(PXR_FIND_TBB_IN_CONFIG "@PXR_FIND_TBB_IN_CONFIG@")
if(PXR_FIND_TBB_IN_CONFIG)
```

or, what about a comment?

```cmake
if(@PXR_FIND_TBB_IN_CONFIG@) # if(PXR_FIND_TBB_IN_CONFIG)
```

Reply:
Did the second option, please let me know if you prefer your original or the comment. 9decb4e0fba8e83f34bbbe0814058ca72e912e1d